examples/argva_node_clustering.py

```diff
@@ -84,52 +75,53 @@ def train():
     loss = model.recon_loss(z, data.train_pos_edge_index)
```

Comment:
missing  loss = loss + model.reg_loss(z) ?

Reply:
This is added in the line below, right?

Reply:
This is fixed in master, see https://github.com/pyg-team/pytorch_geometric/blob/master/examples/argva_node_clustering.py#L77.